odoo/addons/base/models/ir_mail_server.py

```diff
@@ -129,6 +131,14 @@ class IrMailServer(models.Model):
     smtp_debug = fields.Boolean(string='Debugging', help="If enabled, the full output of SMTP sessions will "
                                                          "be written to the server log at DEBUG level "
                                                          "(this is very verbose and may include confidential info!)")
+    max_email_size = fields.Float(string="Max Email Size", default=lambda self: self._get_default_max_email_size(),
```

Comment:
Not sure the field should have a default: more a method that returns the default value if not set (as proposed earlier). That way if you know it you can set it, otherwise just leave it blank and let odoo deal with it.